tasks/release.py

```diff
@@ -16,6 +16,7 @@
 from datetime import date
 from time import sleep
 
+import github
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏‏I think that this will probably make other jobs fail if you don't wrap it with try / except ImportError (see how it is imported in `github_api.py`)

Reply:
switched to a local import instead!